backend/controllers/establishmentRoll/currentlyOut.js

```diff
@@ -0,0 +1,46 @@
+const moment = require('moment')
+
+const {
+  app: { notmEndpointUrl: dpsUrl },
+} = require('../../config')
+
+const { alertFlagLabels } = require('../../shared/alertFlagValues')
+const { putLastNameFirst, stripAgencyPrefix } = require('../../utils')
+
+module.exports = ({ movementsService, logError }) => async (req, res) => {
+  try {
+    const { livingUnitId } = req.params
+    const response = await movementsService.getOffendersCurrentlyOutOfLivingUnit(res.locals, livingUnitId)
+
+    const results = response.currentlyOut
+      .sort((a, b) => a.lastName.localeCompare(b.lastName, 'en', { ignorePunctuation: true }))
+      .map(offender => {
+        const alerts = alertFlagLabels.filter(alertFlag =>
+          alertFlag.alertCodes.some(alert => offender.alerts && offender.alerts.includes(alert))
+        )
+        return {
+          name: putLastNameFirst(offender.firstName, offender.lastName),
+          offenderNo: offender.offenderNo,
+          dob: moment(offender.dateOfBirth, 'YYYY-MM-DD').format('DD/MM/YYYY'),
+          location: stripAgencyPrefix(offender.location, offender.fromAgency),
+          incentiveLevel: offender.iepLevel,
+          currentLocation: offender.toCity,
+          comment: offender.commentText,
+          alerts,
+        }
+      })
+
+    return res.render('establishmentRoll/currentlyOut.njk', {
+      results,
+      livingUnitName: response.location,
+      notmUrl: dpsUrl,
+    })
+  } catch (error) {
+    if (error) logError(req.originalUrl, error, 'Failed to load en route page')
```

Comment:
Is it worth adding a unit test for this bit in future PRs?

Reply:
It definitely won't hurt, I'll add it with the next one.